canadian-journal-economics-default.csl

```diff
@@ -0,0 +1,611 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" demote-non-dropping-particle="never" version="1.0" class="in-text" page-range-format="chicago">
+  <info>
+    <title>Canadian Journal of Economics: InText Default (Author-Date)</title>
+    <id>http://www.zotero.org/styles/canadian-journal-economics-default</id>
```

Comment:
Can you change this to `<id>http://www.zotero.org/styles/canadian-journal-of-economics</id>` (the style file name and "self" link need to be updated to match as well)